lib/builder/generator.js

```diff
@@ -223,9 +223,17 @@ export default class Generator {
       return false
     }
 
+    let minificationOption = this.options.build.htmlMinify
+
+    // Legacy: Override minification options with generate.minify if present
     if (this.options.generate.minify) {
+      minificationOption = this.options.generate.minify
+      consola.warn('generate.minify has been deprecated. Use build.htmlMinify instead!')
```

Comment:
If we all agree to do this way, I suggest adding a message that this option will be removed in 2.1

Reply:
You mean `generate.minify`? Though it's not super clean, I wouldn't remove the override in the next minor version. Otherwise, we don't follow semver.

Reply:
Yeah, what I mean is just making warning message more clear

Reply:
Would `...deprecated and will be removed in the next major version` be sufficient?

Reply:
Yep, good